apps/health_campaign_field_worker_app/lib/utils/utils.dart

```diff
@@ -442,3 +444,18 @@ int getSyncCount(List<OpLog> oplogs) {
 
   return count;
 }
+
+class TotalDetails {
```

Comment:
Remove this or move to dss package utils if needed

Reply:
Removed this unused class